Content.Server/Pinpointer/StationMapComponent.cs

```diff
@@ -0,0 +1,7 @@
+namespace Content.Server.Pinpointer;
+
+[RegisterComponent]
+public sealed class StationMapComponent : Component
```

Comment:
Right now station map doesn't handle grid change. You still see old grid when move player to shuttle.